src/index.js

```diff
@@ -46,6 +46,8 @@ class GooglePlacesSuggest extends Component {
       input: search,
       location: new googleMaps.LatLng(0, 0),
       radius: suggestRadius,
+        types: suggestTypes,
```

Comment:
Please use the same indentation. Use https://github.com/sindresorhus/atom-editorconfig to match the `.editorconfig` file